src/main/java/ReviewResponse.java

```diff
@@ -1,7 +1,17 @@
+import lombok.Data;
+import lombok.Getter;
+import lombok.ToString;
+
+@ToString
+@Data
 public class ReviewResponse {
     private int line;
     private String comment;
 
+    @Getter
+    String okbb;
```

Comment:
Unnecessary field 'okbb' without proper visibility modifier. It should be private to encapsulate the class data. (Severity: medium)